Tests/MapboxNavigationTests/NavigationViewControllerTests.swift

```diff
@@ -333,7 +333,7 @@ class NavigationViewControllerTests: TestCase {
                        newRouteResponse.identifier)
     }
     
-    func testPuck3DLayerPosition() {
+    func disabled_testPuck3DLayerPosition() {
```

Comment:
Let’s track reenabling this test in a separate issue.

Reply:
https://github.com/mapbox/mapbox-navigation-ios/issues/3790